config/routes.rb

```diff
@@ -1,11 +1,24 @@
 Rails.application.routes.draw do
-  root 'pages#hello'
+  devise_for :users, controllers: { confirmations: 'users/confirmations' }
```

Comment:
Please be aware that in your routes file, in the devise_for :users line, you are specifying a custom controller for confirmations as 'users/confirmations', which should be located at `app/controllers/users/confirmations_controller.rb`
Because of this line, when the user clicks the confirmation link from the terminal, this error pops up on the browser:
![image](https://github.com/zhorabay/Budget/assets/83596135/dc9720d7-6649-4f29-a13b-89ca52b01bda)

So please refactor this line of code, to not include the confirmation controller aspect.
It should be as simple as this: 
`devise_for :users`.

Reply:
@nicupop729 Thank you so much!